### PR TITLE
Update version, author and copyright of PDF generated documentation

### PR DESCRIPTION
**Pull request by KygekDev**
Will update https://buildmedia.readthedocs.org/media/pdf/pmmp/rtfd/pmmp.pdf

### CODE REVIEW

**Comment by KygekDev** [comment]
I guess time to open this PR.

File: source/conf.py [hunk 1]
```diff
@@ -204,7 +204,7 @@
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
   ('index', 'PocketMine-MPDocs.tex', u'PocketMine-MP Documentation',
-   u'Intyre', 'manual'),
+   u'dktapps', 'manual'),
```
Comment (dktapps) [inline source/conf.py]:
```suggestion
   u'PocketMine Team', 'manual'),
```

File: source/conf.py [hunk 2]
```diff
@@ -234,7 +234,7 @@
 # (source start file, name, description, authors, manual section).
 man_pages = [
     ('index', 'pocketmine-mpdocs', u'PocketMine-MP Documentation',
-     [u'Intyre'], 1)
+     [u'dktapps'], 1)
```
Comment (dktapps) [inline source/conf.py]:
```suggestion
     [u'PocketMine Team'], 1)
```

File: source/conf.py [hunk 3]
```diff
@@ -248,7 +248,7 @@
 #  dir menu entry, description, category)
 texinfo_documents = [
   ('index', 'PocketMine-MPDocs', u'PocketMine-MP Documentation',
-   u'Intyre', 'PocketMine-MPDocs', 'One line description of project.',
+   u'dktapps', 'PocketMine-MPDocs', 'One line description of project.',
```
Comment (dktapps) [inline source/conf.py]:
```suggestion
   u'PocketMine Team', 'PocketMine-MPDocs', 'One line description of project.',
```

**Review by dktapps — CHANGES_REQUESTED** [review]

**Comment by KygekDev** [comment]
I have committed all of your requested changes.

**Comment by KygekDev** [comment]
Thanks.

**Pull request merged**